src/components/Search/SearchResultSeries.js

```diff
@@ -0,0 +1,158 @@
+          key={s.id}
+          onClick={() => this.logClick(...logLinkParams)}
+          to={`/lessons/series/c/${s.id}`}
+        >
+          <span className="margin-right-8 margin-left-8">
+            <Icon name="tasks" size="small" />
+            {`${t('search.showAll')} ${s.cuIDs.length} ${t('pages.collection.items.lessons-collection')}`}
+          </span>
+        </Link>
+      </Button>
+    );
+  };
+
+  buildCollectionLinkParams = c => {
+    const { queryResult: { search_result: { searchId } }, hit, rank, filters }      = this.props;
+    const { _index: index, _source: { mdb_uid: mdbUid, result_type: resultType }, } = hit;
+
+    return {
+      canonicalLinkParams: [{ id: c.id, content_type: 'CT_LECTURE_SERIES' }, this.getMediaLanguage(filters)],
+      logLinkParams: [mdbUid, index, resultType, rank, searchId]
+    };
+  };
+
+  getLowestLevelSeries = (series, rootId) => {
+    if (series.length <= 1)
+      return series[0];
+    const root = series.find(s => s.parent_id === rootId || (!s.parent_id && !rootId));
+    if (!(root?.children?.length !== 0)) {
+      return root;
+    }
+
+    return this.getLowestLevelSeries(series, root.id);
+  };
+
+  handleToggle = () => this.setState({ showAll: !this.state.showAll });
+
+  render() {
+    const {
+      t,
+      location,
+      hit,
+      getSerieBySource,
+      getSerieByTag,
+      nestedDenormCollectionWUnits,
+      getTagById,
+      wip: { lectures: wipL, series: wipS }
+    } = this.props;
+
+    const { showAll } = this.state;
+
+    if (wipL || wipS) {
+      return null;
+    }
+
+    const isByTag  = hit._type === 'lessons_series_by_tag';
+    const getSerie = isByTag ? getSerieByTag : getSerieBySource;
+
+    const { _score: score, _uid, _source: { mdb_uid } } = hit;
+    if (!_uid) {
+      return null;
+    }
+
+    const series = _uid.split('_').map(getSerie);
+    const s      = this.getLowestLevelSeries(series);
+    if (!s) return null;
+    if (s.collections.length === 1) {
+      const c = nestedDenormCollectionWUnits(s.collections[0].id);
+      return (
+        <SearchResultCollection c={c} {...this.props} />
+      );
+    }
+
+    const { logLinkParams } = this.buildLinkParams();
+    const title             = isByTag ? getTagById(_uid)?.label : s.name;
+    const collecitons       = s.collections.filter(c => c.id !== mdb_uid);
+    collecitons.unshift(s.collections.find(c => c.id === mdb_uid));
+    return (
+      <Segment className="bg_hover_grey search__block">
+        <Container>
+          <Container>
+            <Container as="h3" content={title} />
+            <Container className="content">
+              {this.iconByContentType(CT_LESSONS_SERIES, t, true)}
+            </Container>
+            <div className="clear" />
+          </Container>
+
+          <Container className="content clear margin-top-8">
+            {(showAll || collecitons.length < 5 ? collecitons : collecitons.slice(0, 5)).filter(c => !!c).map(this.renderSerie)}
```

Comment:
What is the order of collections? Why?

Reply:
its default order of collections from mdb (by film_date ...) except only that was find with elastic. This one I push first